package-lock.json

```diff
@@ -27611,7 +27611,7 @@
     },
     "packages/web3wallet": {
       "name": "@walletconnect/web3wallet",
-      "version": "2.12.1",
```

Comment:
Curious: how did this end up with the wrong version and needed to be corrected?

Reply:
it was me setting `private` to `false` and having to revert https://github.com/WalletConnect/walletconnect-monorepo/pull/4406/commits/dbdc6555714864d18e818a91c4a4cb7c81cab723